Rubberduck.RibbonDispatcher/AbstractRibbonDispatcher/RibbonCommon.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using stdole;
+using Microsoft.Office.Core;
+
+namespace RubberDuck.RibbonDispatcher {
+    using ControlSize         = RibbonControlSize;
+    using LanguageStrings     = IRibbonTextLanguageControl;
+    using ChangedEventHandler = EventHandler<ChangedControlEventArgs>;
+
+    public class RibbonCommon : IRibbonCommon {
```

Comment:
Class should be `abstract` if it's not intended to be instantiated directly; accordingly, name would be something like `RibbonControlBase`.

Reply:
Ribbon Groups use direct instances of this at the moment, as ALL of their functionality in encompassed. However there could be a bare concrete class around a base abstract class instead.